components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/v1/dto/ApplicationDTO.java

```diff
@@ -294,12 +313,13 @@ public boolean equals(java.lang.Object o) {
         Objects.equals(subscriptionCount, application.subscriptionCount) &&
         Objects.equals(keys, application.keys) &&
         Objects.equals(attributes, application.attributes) &&
-        Objects.equals(subscriptionScopes, application.subscriptionScopes);
+        Objects.equals(subscriptionScopes, application.subscriptionScopes) &&
+        Objects.equals(owner, application.owner);
   }
 
   @Override
   public int hashCode() {
-    return Objects.hash(applicationId, name, subscriber, throttlingPolicy, description, tokenType, status, groups, subscriptionCount, keys, attributes, subscriptionScopes);
+    return Objects.hash(applicationId, name, subscriber, throttlingPolicy, description, tokenType, status, groups, subscriptionCount, keys, attributes, subscriptionScopes, owner);
```

Comment:
Exceeding the max length of a line. Shall we fix this formatting issue?

Reply:
changed and resolved